File: src/test/java/com/lsd/SanitiserTest.java
package com.lsd;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class SanitiserTest {

    @Test
    void handleNullValuesSafely() {
        assertThat(Sanitiser.sanitise(null)).isEqualTo("");
    }

    @Test
    void removesPlantUmlMarkup() {
        assertThat(Sanitiser.sanitise("input <$something>")).isEqualTo("input ");
    }

    @Test
    void trimLeadingWhitespace() {
        assertThat(Sanitiser.sanitise(" abc")).isEqualTo("abc");
    }

    @Test
    void trimTrailingWhitespace() {
        assertThat(Sanitiser.sanitise("abc ")).isEqualTo("abc");
    }
}

File: src/test/kotlin/com/lsd/SanitiserTest.kt
package com.lsd

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

internal class SanitiserTest {
    @Test
    fun handleNullValuesSafely() {
        Assertions.assertThat(sanitise(null)).isEqualTo("")
    }

    @Test
    fun removesPlantUmlMarkup() {
        Assertions.assertThat(sanitise("input <\$something>")).isEqualTo("input ")
    }

    @Test
    fun trimLeadingWhitespace() {
        Assertions.assertThat(sanitise(" abc")).isEqualTo("abc")
    }

    @Test
    fun trimTrailingWhitespace() {
        Assertions.assertThat(sanitise("abc ")).isEqualTo("abc")
    }
}